examples/dt/neutron/CRD_files/neutron_az_ovn.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  neutron:
+    template:
+      customServiceConfig: |
+        [DEFAULT]
+        dns_domain = openstackgate.local
+        vlan_transparent = true
+        network_scheduler_driver = neutron.scheduler.dhcp_agent_scheduler.AZAwareWeightScheduler
+        dhcp_load_type = networks
+        default_availability_zones = zone-1,zone-2
+        igmp_snooping_enable = true
```

Comment:
Duplicate of line 19, so can be dropped from here as this belong to ovs group

Also i don't see it's mentioned in dt file, from where it's coming?

Reply:
This line will be dropped as it is found only in [ovs] section of neutron.conf.

Should this be mentioned in DT ?